examples/amp-access-fewcents.html

```diff
@@ -124,6 +121,7 @@
       }
     </style>
     <script async custom-element="amp-access" src="https://cdn.ampproject.org/v0/amp-access-0.1.js"></script>
+    <script async custom-element="amp-access-fewcents" src="https://cdn.ampproject.org/v0/amp-access-fewcents-0.1.js"></script>
```

Comment:
@ampproject/wg-caching can you tell why this would cause the validation checks to throw:

```
examples/amp-access-fewcents.html:124:4 Custom JavaScript is not allowed. (see https://amp.dev/documentation/guides-and-tutorials/learn/validation-workflow/validation_errors/#custom-javascript-is-not-allowed)
```

The validation config included in this PR seems like it should allow this, right?

Reply:
I was actually looking into this a bit with @twifkak and @ychsieh on a different PR on Friday. It seems  `amp validate-html-fixtures` eventually calls the `amphtml-validator ` npm package https://github.com/ampproject/amphtml/blob/624997ab9bbbd839daaeb84791ad3702278239e8/build-system/tasks/validate-html-fixtures.js#L91

I am assuming that 
1. checking in the `protoascii` file in a pr 
2. build new npm package in validator repo
3. update package.json in amp
4. check in the example 

would work but seems quite tedious. Given @twifkak nor I remember having this issue in the past, perhaps I am missing something. Maybe @rsimha has some insight about how this check is supposed to work with a brand new extension & example?

Reply:
Does `amphtml-validator` contain a static copy of the protoascii files, and is therefore not seeing the new changes in this PR? If yes, is there a way to (say, via a flag) make it use the local protoascii files instead of the ones in the npm package?

Failing this, you could add the new html file to this allowlist to start with, and when the protoascii changes land in a new version of `amphtml-validator`, remove the file from the allowlist.

https://github.com/ampproject/amphtml/blob/624997ab9bbbd839daaeb84791ad3702278239e8/build-system/test-configs/config.js#L182-L205

Reply:
Yeah, took me a bit to get up to speed with this. I chased a few wild geese on Friday. I agree, it seems like the best fix is to update the `validate-html-fixtures` check to use a custom-built engine from the local code.

The amphtml-validator is a thin shell around the actual validator engine, which it pulls from `cdn.ampproject.org` (and which embeds a copy of the rules at build time). There is a flag to override the engine: https://github.com/ampproject/amphtml/blob/2c41c9556561c4072f46595cb78faba5af307147/validator/js/nodejs/index.js#L382-L388

And here are [instructions for building](https://github.com/ampproject/amphtml/tree/main/validator#building-validator-engine) the new WASM engine.

Reply:
Thanks folks! I created an issue to track the future improvements here https://github.com/ampproject/amphtml/issues/37467

@mayank-s-dev for now I think you have two options:
1. update the `build-system/test-configs/config.js` file Raghu linked above and we can remove it when the validator package gets updated.
2. Remove the `examples` file from this PR and check it in in a follow up PR once validator is updated.

Reply:
@calebcordry @twifkak @rsimha Thank you guys for your help.

Caleb I have updated the `build-system/test-configs/config.js` file, and added fewcents file to the allowed list.

Reply:
@newmuis @calebcordry Error related to the example file has been resolved.